Documentation/Conditions/Reference/Index.rst

```diff
@@ -1009,3 +1009,40 @@ shows how quotes can be escaped::
 
     [userFunc = user_testFunctionWithThreeArgumentsEscapedQuotes(1, 2, "3, \"4, 5\", 6")]
 
+
+.. _condition-site:
+
+site
+====
+
+Syntax::
+
+   [site = identifier = someIdentifier, base = https://www.typo3.org/]
+
+Comparison: The identifier of the site name is evaluated.
```

Comment:
What I'm missing is an information about the `base`. What is `base` in this syntax, what are possible values?